feat(daemon): Support spinnaker-monitoring-local.yml

ewiseblatt opened the pull request.

jfillo commented:
LGTM 
+1 for the unit test

jfillo commented:
Actually, you should add the test case where the '-local.yml' doesn't exist. Its a simple case, but should be included for completeness sake.

ewiseblatt commented:
I added another test (and re-ordered the ones there). Sorry I already
squashed it.
All that changed was the test for without override.

On Wed, Nov 21, 2018 at 11:44 AM Jason Fillo <notifications@github.com>
wrote:

> Actually, you should add the test case where the '-local.yml' doesn't
> exist. Its a simple case, but should be included for completeness sake.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spinnaker/spinnaker-monitoring/pull/123#issuecomment-440733559>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEKkGAZ2p_qIwIr21qmhohKEOWHRFRacks5uxYLmgaJpZM4YtdnH>
> .
>
